doc/dev/release.md

```diff
@@ -45,6 +45,24 @@ $ git config [--global] user.signingkey "$GPG_KEY_ID"
 $ git config [--global] user.email "$GPG_EMAIL"
 ```
 
+Also, make sure that you setup the git gpg config as follows.
+```bash
+ cat ~/.gnupg/gpg.conf
```

Comment:
Why run this command?

Reply:
To check if the config files are set up with the `default-key`

Reply:
```suggestion
$ cat ~/.gnupg/gpg.conf
...
```